IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/PropertySets/Pset_LinearReferencingMethod/DocPropertySet.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_LinearReferencingMethod" UniqueId="a4db95cf-06af-4562-8d73-bbd65fbcc8da" ApplicableType="IfcAlignment,IfcReferent/(.POSITION.)">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_LinearReferencingMethod" UniqueId="a4db95cf-06af-4562-8d73-bbd65fbcc8da" ApplicableType="IfcAlignment,IfcReferent/POSITION" PropertySetType="PSET_OCCURRENCEDRIVEN">
```

Comment:
Need to check if this could be on any kind of IfcReferent => Discuss!

Reply:
@SergejMuhic

Reply:
Any kind of `IfcReferent`.